src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-mon calendar--month-length-29">
```
<img width="853" alt="Screenshot 2023-10-27 at 14 56 50" src="https://github.com/mate-academy/layout_calendar/assets/135232001/c1ed3e90-3f8c-4104-933e-a18b300e4296">